deeppavlov/models/embedders/fasttext_embedder.py

```diff
@@ -59,6 +59,9 @@ def __init__(self, load_path: Union[str, Path], save_path: Union[str, Path] = No
         self.pad_zero = pad_zero
         self.model = self.load()
 
+    def destroy(self):
+        del self.model
```

Comment:
`def destroy` is there twice now because of a merge conflict, it seems

Reply:
I don't see any conflict.